recipe/meta.yaml

```
@@ -23,10 +23,9 @@ outputs:
     requirements:
       run:
         - cuda-nvcc_{{ target_platform }} {{ version }}.*
-    test:
-      requires:
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
```

You probably want to do `{{ c_compiler }}` instead to avoid pinning a version.

---

Thanks for the feedback! 🙏

Yeah that seems reasonable

Should we do [the same thing in the `compilers-feedstock`]( https://github.com/conda-forge/compilers-feedstock/blob/c4865f325ba752842f7ea21470abde447f7f1062/recipe/meta.yaml#L38 )?

---

Nope. The purpose of `compilers` feedstock is to give the users the same compilers used by conda-build, whereas `cuda-nvcc` is a way to give users nvcc.

---

Though we would like one that plays well with the same compilers conda-build uses. So maybe it is not that far off

What if we did this?

```suggestion
        - c-compiler
        - cxx-compiler
```

---

Just to update this thread, as `cuda-compiler` fills a similar purpose to `c-compiler` & `cxx-compiler` (and [includes them as dependencies]( https://github.com/conda-forge/cuda-compiler-feedstock/blob/daa71ce4a23bac73c193f3a3794c6e6c4e7261dc/recipe/meta.yaml#L15-L16 )), think Isuru's original suggestion is probably the best choice here